state/subnets.go

```diff
@@ -355,18 +354,17 @@ func (st *State) AddSubnet(args network.SubnetInfo) (subnet *Subnet, err error)
 }
 
 func (st *State) addSubnetOps(id string, args network.SubnetInfo) (subnetDoc, []txn.Op, error) {
-	// TODO (hml) 2019-07-24
-	// Temporary until SubnetInfo.SpaceName is changed to SubnetInfo.SpaceID
 	unique, err := st.uniqueSubnet(args.CIDR, string(args.ProviderId))
 	if err != nil {
 		return subnetDoc{}, nil, errors.Trace(err)
 	}
 	if !unique {
 		return subnetDoc{}, nil, errors.AlreadyExistsf("subnet %q", args.CIDR)
 	}
-	sp, err := st.Space(args.SpaceName)
-	if err != nil {
-		return subnetDoc{}, nil, errors.Trace(err)
+	if args.SpaceID == "" && args.SpaceName == network.DefaultSpaceName {
+		// Ensure the subnet is added to the default space
+		// if none is defined for the subnet.
+		args.SpaceID = "0"
```

Comment:
There should be a `network.DefaultSpaceID` constant we can use here.

Reply:
agreed